contracts/modules/RelayerManager.sol

```diff
@@ -352,12 +359,16 @@ abstract contract RelayerManager is BaseModule {
                         require(whitelistAfter > 0 && whitelistAfter < block.timestamp, "RM: refund not authorised");
                     }
             }
-            uint256 gasConsumed = _startGas.sub(gasleft()).add(14000);
-            uint256 refundAmount = Utils.min(gasConsumed, _gasLimit).mul(_gasPrice);
+            uint256 refundAmount;
             if (_refundToken == ETH_TOKEN) {
+                uint256 gasConsumed = _startGas.sub(gasleft()).add(14000);
```

Comment:
Where has the 14K value come from? Maybe update the pr description with that. The logic we had before was dependent on the number of signatures, i.e. 30k/40k

Reply:
It was dependent on the use or not of the daily limit, which we determined using the  number of signatures. This is no longer relevant.

Now the cost of the remaining logic (i.e. `Utils.min()` twice, `invokeWallet` and emitting the `Refund` event) is constant and estimated at 14k gas when the refund is in ETH based on existing mainnet transactions. When the refund is in ERC20 we need to increase that number by 14500 gas (10k to update the 2 balances on the ERC20 contract and 4.5k to get the token price from the UniswapV2 pool).

I'll add a card to make sure we verify these numbers when testing on Ropsten and Mainnet.